lib/tasks/whitehall_news_importer.rb

```diff
@@ -31,6 +31,8 @@ def create_or_update_document(translation, edition, document)
         review_state: "unreviewed",
         summary: translation["summary"],
         tags: tags(edition),
+        has_live_version_on_govuk: edition["state"] == "published",
```

Comment:
Is this correct? `has_live_version_on_govuk` should be true if there was _ever_ a live version of the document, not just if the edition was published.

Reply:
Yeah, I get that - I was trying to replicate the way we're setting it in the app at the moment. But I'm happy to write a separate card to backfill doing this on the import.